cpu/stm32f0/periph/adc.c

```diff
@@ -92,7 +92,8 @@ int adc_sample(adc_t line,  adc_res_t res)
     prep();
 
     /* set resolution and channel */
-    ADC1->CFGR1 = res;
+    ADC1->CFGR1 &= ~(ADC_CFGR1_RES);
+    ADC1->CFGR1 |= res;
```

Comment:
I dont quite get the benefit of this change, as I see it, it does the same thing, just spending 5 more cpu cycles for it...